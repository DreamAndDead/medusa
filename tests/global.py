foo = 42

def bar():
    global foo
    foo = 34

print("foo = ", foo)
bar()
print("foo = ", foo)