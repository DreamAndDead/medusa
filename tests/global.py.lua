dofile('./pythonlua/luainit.lua')

local foo = 42
local function bar()
    foo = 34
end
print("foo = ", foo)
bar()
print("foo = ", foo)