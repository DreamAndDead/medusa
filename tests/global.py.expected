foo = 	42
foo = 	34
